src/test/kotlin/firstweekstringcalculator/StringCalculator.kt

```diff
@@ -0,0 +1,66 @@
+package firstweekstringcalculator
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.Test
+
+class StringCalculator {
+
+    @Test
+    fun stringCalculatorTestFunction() {
+        val calculationString: String = "2 + 3 * 4 / 2"
+        val stringCalculation = StringCalculation()
+
+        assertThat(stringCalculation.calculate(calculationString)).isEqualTo(10.0)
+    }
+
+    @Test
+    fun ariseStringCalculatorIllegalArgumentExceptionCaseNull() {
+        val calculationString: String? = null
+        val stringCalculation = StringCalculation()
+
+        assertThatThrownBy { stringCalculation.calculate(calculationString) }
+            .isInstanceOf(IllegalArgumentException::class.java)
+            .hasMessageContaining("입력값이 Null 이거나 공백일 수 없습니다")
+    }
+
+    @Test
+    fun ariseStringCalculatorIllegalArgumentExceptionCaseBlank() {
+        val calculationString: String = "  "
+        val stringCalculation = StringCalculation()
+
+        assertThatThrownBy { stringCalculation.calculate(calculationString) }
+            .isInstanceOf(IllegalArgumentException::class.java)
+            .hasMessageContaining("입력값이 Null 이거나 공백일 수 없습니다")
+    }
+
+    @Test
+    fun ariseStringCalculatorIllegalArgumentExceptionCaseWrongOperator() {
+        val calculationString: String = "2 $ 3 * 4 / 2"
+        val stringCalculation = StringCalculation()
+
+        assertThatThrownBy { stringCalculation.calculate(calculationString) }
+            .isInstanceOf(IllegalArgumentException::class.java)
+            .hasMessageContaining("연산자로 사용하실 수 없습니다. +, -, *, / 의 연산을 사용해주세요.")
```

Comment:
```suggestion
        assertThatThrownBy { stringCalculation.calculate(calculationString) }
            .isInstanceOf(IllegalArgumentException::class.java)
            .hasMessage("$ 를 연산자로 사용하실 수 없습니다. +, -, *, / 의 연산을 사용해주세요.")
```
hasMessage를 이용해서 더 정확한 테스트를 진행해보자.